thenewboston_node/business_logic/docs/templates/blockchain-format.rst

```diff
@@ -67,7 +66,7 @@ In-memory MessagePack_ compacted object === `Uncompact`_ ==> In-memory MessagePa
 Decompress
 ^^^^^^^^^^
 
-Just use corresponding decompression algorithm. Here is how we do it in Python::
+Use corresponding decompression algorithm. For example, in Python use:
```

Comment:
@led76marx this is the reason for failing tests. Double colon (::) is special RST syntax.